org.gtk.oomox.json

```diff
@@ -0,0 +1,128 @@
+{
+    "app-id": "org.gtk.oomox",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "oomox-gui",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--filesystem=~/.themes",
+        "--filesystem=~/.icons",
+        "--socket=session-bus",
+        "--socket=x11"
+    ],
+    "modules": [
+        {
+            "name": "sassc",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cd ./deps/libsass ; autoreconf -i ; ./configure --prefix=/app ; make ; make DESTDIR=/ install",
```

Comment:
This looks like a standard build, you don't need to use custom commands. You can set a `subdir` if that is needed.

Reply:
their buildsystem can rely on filetree, so if libsass is one lever upper sassc will link to it

Reply:
Alright, acceptable.